### ISSUE TITLE
Spacing issue

### ISSUE THREAD

**Issue by calebthecurious** [calebthecurious/react-portfolio-2022#1]
I have a styling issue that can be solved by fixing each scene/page to its respective position. Please see below for examples of spacing issues. 

**Landing Page**
Spacing between  _**landing page**_ and _**skills page**_  overlaps when the screen size is 1064 x 787. How do I lock these components so that they stay in place and do not move reactively with the screen being made larger vertically?

![image](https://user-images.githubusercontent.com/59587190/220733652-da755418-891e-4e9e-954c-04ac10d2cdf9.png)

**Project section bleeding into skills section**
within 
1060px and 1212px wide - I believe lg screen size?, the skills section and the project section bleed overlap. This has been extremely frustrating to figure out how to fix.  
![image](https://user-images.githubusercontent.com/59587190/220732125-bf0188f1-05d5-4585-9ff5-e66c7ad3c3ba.png)

**Testimonal Section**
The " speech marks almost disappear with the small screen and they are the spacing is too big when the screen size is full. 

![image](https://user-images.githubusercontent.com/59587190/220732210-3b4c80dd-cc13-459f-9db8-ca6adf59b878.png)
![image](https://user-images.githubusercontent.com/59587190/220733131-7f9b8167-9e18-4702-acae-5cf737af4a50.png)

**Contact Section**
There is a lot of empty space as seen between the contact form images attached below. 

![image](https://user-images.githubusercontent.com/59587190/220732306-25790593-138f-4b01-a061-c975d53fbe2a.png)
![image](https://user-images.githubusercontent.com/59587190/220732399-c0e436db-1c5d-4e6a-911a-3ce92869db78.png)

**Comment by jcheunga** [comment]
Seems to be good now or is there something in this issue that needs addressing.

**Comment by calebthecurious** [comment]
Yeah, I think that fixed the overlapping issue, however, there's still a lot of space between each scene. I'm wondering why each section's padding is not reflective of the responsiveness of the screen size but rather the responsive to specific page size. 🤔

**Comment by jcheunga** [comment]
You need to use media queries, eg md-, lg-
https://tailwindcss.com/docs/responsive-design
